STACpopulator/api_requests.py

```diff
@@ -0,0 +1,51 @@
+import os
```

Comment:
add `class bcolors`  that is in .depecrated/collection_processor.py since it has not been defined and is used in `post_stac_collection`